destinyitemmanager.com/index.html

```diff
@@ -1,167 +1,177 @@
-          <a class="icon-button" target="_blank" href="https://twitter.com/ThisIsDIM">
-            <img src="images/icons_dTwitter.png" alt="Twitter">
-          </a>
-          <a class="icon-button" target="_blank" href="https://www.reddit.com/r/DestinyItemManager/">
-            <img src="images/icons_dReddit.png" alt="Reddit">
-          </a>
-          <a class="icon-button" target="_blank" href="https://github.com/DestinyItemManager/DIM/">
-            <img src="images/icons_dGithub.png" alt="GitHub">
-          </a>
-          <a class="icon-button" target="_blank" href="https://discordapp.com/invite/UK2GWC7">
-            <img src="images/icons_dDiscord.png" alt="Discord">
-          </a>
-        </div>
+          </g>
+        </svg>
       </div>
-      <div class="row">
-        <div class="col-6">
-          <ul class="footer-bottom-nav">
-            <li>Destiny is a registered trademark of Bungie. Some content and images are the property of Bungie.</li>
-          </ul>
-        </div>
-        <div class="col-6">
-          <div class="copy">&copy; 2017 DIM</div>
-        </div>
+      <div class="col-6 nav">
+        <ul>
+          <li class="link-headings">Useful Links</li>
+          <li><a class="link-text" target="_blank" href="https://github.com/DestinyItemManager/DIM/issues">Support</a></li>
+          <li><a class="link-text" href="https://beta.destinyitemmanager.com">Beta</a></li>
+          <li><a class="link-text" href="https://app.destinyitemmanager.com">Launch App</a></li>
+          <li><a class="link-text" target="_blank" href="https://twitter.com/ThisIsDIM">Contact</a></li>
+          <li><a class="link-text" target="_blank" href="https://timesavedwith.destinyitemmanager.com/">Time Saved with
+              DIM</a></li>
+        </ul>
+      </div>
+      <div class="col-4 social">
+        <a class="icon-button" target="_blank" href="https://twitter.com/ThisIsDIM">
+          <img src="images/icons_dTwitter.png" alt="Twitter">
+        </a>
+        <a class="icon-button" target="_blank" href="https://www.reddit.com/r/DestinyItemManager/">
+          <img src="images/icons_dReddit.png" alt="Reddit">
+        </a>
+        <a class="icon-button" target="_blank" href="https://github.com/DestinyItemManager/DIM/">
+          <img src="images/icons_dGithub.png" alt="GitHub">
+        </a>
+        <a class="icon-button" target="_blank" href="https://discordapp.com/invite/UK2GWC7">
+          <img src="images/icons_dDiscord.png" alt="Discord">
+        </a>
       </div>
-    </footer>
-    <div class="sliding-panel-fade-screen"></div>
-    <script src="scripts/main.js"></script>
-  </body>
-</html>
+    </div>
+    <div class="row">
+      <div class="col-6">
+        <ul class="footer-bottom-nav">
+          <li>Destiny is a registered trademark of Bungie. Some content and images are the property of Bungie.</li>
+        </ul>
+      </div>
+      <div class="col-6">
+        <div class="copy">&copy; 2017 DIM</div>
```

Comment:
2018!

Reply:
I wasn't sure about that, do you change the year every year on copyright? or is it the year it was made?